src/dump.cpp

```diff
@@ -157,6 +160,13 @@ void game::dump_stats( const std::string& what, dump_mode mode )
             r.push_back( veh_empty.name );
             r.push_back( to_string( veh_empty.total_mass() ) );
             r.push_back( to_string( veh_fueled.total_mass() ) );
+            r.push_back( to_string( veh_fueled.max_velocity() ) );
+            r.push_back( to_string( veh_fueled.safe_velocity() ) );
```

Comment:
Might be best formatted to fewer significant figures, possibly also a percentage?